### PR TITLE
Improve documentation of Zope's Error Logging Services

### PR DESCRIPTION
**Pull request by jugmac00**
SiteErrorLog can be configured to ignore certain exceptions.

Now, this is also documented.

cf #671
cf https://github.com/zopefoundation/Products.PluggableAuthService/issues/53

modified:   CHANGES.rst
modified:   docs/zopebook/ZopeServices.rst

### CODE REVIEW

File: docs/zopebook/ZopeServices.rst [hunk 1]
```diff
@@ -340,25 +340,32 @@ Now you can use the new MailHost object from a Script.
...
-under the name 'error_log', provides debugging and error logging
-information in real-time.  When your site encounters an error, it
-will be logged in the Site Error Log, allowing you to review (and
-hopefully fix!) the error.
+.. note::
 
-Options settable on a Site Error Log instance
-include:
+  As of Zope 4, the **Site Error Log** is no longer a a dependency of Zope,
+  but has to be installed additionally if you want to use it.
```
Comment (dataflake) [inline docs/zopebook/ZopeServices.rst]: Replace „additionally“ with „separately“

File: docs/zopebook/ZopeServices.rst [hunk 2]
```diff
@@ -340,25 +340,32 @@ Now you can use the new MailHost object from a Script.
...
 
-Number of exceptions to keep
-  keep 20 exceptions by default, rotating "old" exceptions out when more
-  than 20 are stored.  Set this to a higher or lower number as you like.
+The **Site Error Log** object, typically accessible in the Zope root
+under the name `error_log`, provides debugging and error logging
+information in real-time. When your site encounters an error, it
```
Comment (dataflake) [inline docs/zopebook/ZopeServices.rst]: „real time“ - no hyphen

File: docs/zopebook/ZopeServices.rst (hunk 1 above)
Comment (icemac) [reply docs/zopebook/ZopeServices.rst]: Do we have a section which describes how to install Zope Products?
At least the actual name of the package (`Products.SiteErrorLog`) should be mentioned.

Comment (jugmac00) [reply docs/zopebook/ZopeServices.rst]: @dataflake Thanks!

@icemac Added the package name and added a link to PyPi.

Yes, there is section which explains how to install a product, but.. oh boy... this needs some **polish**.

https://zope.readthedocs.io/en/latest/zopebook/MaintainingZope.html#installing-new-products

File: docs/zopebook/ZopeServices.rst (hunk 2 above)
Comment (jugmac00) [reply docs/zopebook/ZopeServices.rst]: Good catch!

**Pull request merged**